docs/scripting-ref.rst

```diff
@@ -2230,3 +2230,128 @@ Trigger mouse release event in web page.
 **Async:**: no.
 
 See notes about mouse events in :ref:`splash-mouse-click`.
+
+.. _splash-send-keys:
+
+splash:send_keys
+----------------
+
+Send keyboard events to page context.
+
+**Signature:** ``splash:send_keys(keys)``
+
+**Parameters**
+
+* keys - string representing the keys to be sent as keyboard events.
+
+**Returns:** nil
+
+**Async:**: no.
+
+Key sequences are specified by using a small subset of emacs edmacro syntax:
+
+* whitespace is ignored and only used to separate the different keys
+* characters are literally represented
+* ``RET``, ``SPC``, ``TAB``, ``DEL``, ``ESC`` are reserved and represent
+  special control characters.
+* words within brackets represent function keys, like ``<Return>``, ``<Left>``
+  or ``<Home>``. See `Qt docs`__ for a full list of function keys. ``<Foo>``
+  will try to match ``Qt::Key_Foo``.
```

Comment:
edmacro format *is* repetitive, there are more ways to do smth, with Return key being one of the richest having `RET`, `<return>`, `^M`, `C-m` and `\015` as alternative representations. I suppose those shorthand notations were selected as one of the most frequent ones to be used.